Cargo.toml

```diff
@@ -9,3 +9,6 @@ members = [
 exclude = [
     "ci/no-std-check",
 ]
+
+[patch.crates-io]
```

Comment:
Just FYI, ibc-proto-rs v0.22.0 has now been released (with these changes) so we can now depend on it directly. https://github.com/cosmos/ibc-proto-rs/releases/tag/v0.22.0